zabbix/proxy/init.sls

```diff
@@ -21,6 +21,7 @@ zabbix-proxy:
       - pkg: zabbix-proxy
       - file: zabbix-proxy-logdir
       - file: zabbix-proxy-piddir
+      - file: zabbix-proxy-sqlitedir
```

Comment:
If `zabbix-proxy-sqlitedir` will not be rendered due failing `if` condition, whole state will fail due missing requisite, therefore I add `watch_in` requisite in `zabbix-proxy-sqlitedir` itself.
```suggestion
```